extension/src/experiments/model/collect.ts

```diff
@@ -316,22 +316,32 @@ const formatCommitMessage = (commit: string) => {
   return `${lines[0]}${lines.length > 1 ? ' ...' : ''}`
 }
 
+const getCommitMessages = (commitsOutput: string) => {
+  const commits = commitsOutput.split('\u0000').map(commit => {
```

Comment:
[Q] Could this be a constant imported from `extension/src/cli/git/constants.ts`?

Reply:
This being `'\u0000'`